apps/web/cypress/tests/digest-playground.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('Digest Playground Workflow Page', function () {
     cy.url().should('include', '/digest-playground');
     cy.contains('Digest Workflow Playground');
 
-    cy.get('a[href="https://docs.novu.co/workflows/digest?utm_campaign=in-app"]').contains('Learn more in docs');
+    cy.get('a[href="https://docs.novu.co/workflows/digest?utm_campaign=inapp-digest"]').contains('Learn more in docs');
```

Comment:
It's quite uncomfortable for test maintainability to be including marketing parameters in the test queries. Our implementation logic (and therefore testing) shouldn't change when we modify the marketing parameters.

We can update the query parameter like so to remove the dependency on the UTM query param, the `=^` acts as a "string starts with" matcher.
```suggestion
    cy.get('a[href^="https://docs.novu.co/workflows/digest"]').contains('Learn more in docs');
```

Reply:
I didn't realize that was an actual part of the test suite. Lesson learned!